### PR TITLE
Arabic Omelette

### PR DESCRIPTION
**Pull request by Karyum**
rel #8

I can't find any pics for the Arabic Omelette on my phone if anyone has a picture of it please add it to the PR thanks!

### CODE REVIEW

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: How would I know the amount the first time I make this. You mix everything together and I would rather not taste a raw egg mixture to check the taste

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: Change stir to whisk :man_cook:

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: How much is a bit?

Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: Typo *until

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: Might be a nice idea to add in an ingredient combination that has worked well for you? :+1:

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: Either capitalise everything or don't capitalise at all :maple_leaf:

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: Why is fork **bold**?

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: I would change the wording of this slightly:
> it is up to you how much you want your omelette to be cooked but I would recommend flipping when its golden-red

Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: is golden red even a colour? :upside_down_face:

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [inline recipes/breakfest/arabic_omelette.md]: I would maybe say the difficulty is _slightly_ higher, like it is super easy but requires a teeny bit of attention - I would say 10% is literally like a bowl of cereal

Comment (m4v15) [reply recipes/breakfest/arabic_omelette.md]: Happy to go with a consensus on this tho

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [inline recipes/breakfest/arabic_omelette.md]: Bad english

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [inline recipes/breakfest/arabic_omelette.md]: steps 2-5 should be capitalised, as per step 1

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [inline recipes/breakfest/arabic_omelette.md]: lose the full stops?

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [inline recipes/breakfest/arabic_omelette.md]: I don't think Mrs Saliba invented omelettes...

**Review by m4v15 — CHANGES_REQUESTED** [review]

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: Omelette is spelt wrong also shouldn't be capitalised

File: README.md
Comment (matthewdking) [inline README.md]: Omelette is spelt wrong

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfest/arabic_omelette.md]: Omelette

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: yeah good point

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: Mate first of all it's omelettes and not omelletes.
send of all she is the author and not the inventor

Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: @Karyum omelette is spelt incorrectly four times in the recipe

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: you just being a dick here mate

File: recipes/breakfest/arabic_omelette.md
Comment (finnhodgkin) [inline recipes/breakfest/arabic_omelette.md]: I wouldn't say the reception I witnessed was _very_ positive. Positive maybe.

I should really get my review app in order so we can get some anonymous feedback.

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: I'm sorry are you color blind ? do you even color mate ? have you seen the color gold ? have you seen the color red ?
connect the dots

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: I literally said i want people to be creative i should not give hints on what to add

**Review by finnhodgkin — CHANGES_REQUESTED** [review]
Breakfest??

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [reply recipes/breakfest/arabic_omelette.md]: tbf golden brown is the more common phrase - and also more appropriate in this case maybe?

Comment (m4v15) [reply recipes/breakfest/arabic_omelette.md]: >I would change the wording of this slightly:

@matthewdking 
I think this comment isn't entirely useful without actually suggesting how or why mate 😕

Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: good point mate i trust you @m4v15 bless you

Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: @m4v15 I agree but that's me using bad markdown! That quote in my review comment is what I think it should be updated to :+1: open to other suggestions though

File: recipes/breakfest/arabic_omelette.md
Comment (matthewdking) [reply recipes/breakfest/arabic_omelette.md]: Fair point :shipit:

File: recipes/breakfest/arabic_omelette.md
Comment (m4v15) [reply recipes/breakfest/arabic_omelette.md]: @matthewdking oh I see, yeah I thought you were just quoting it

File: recipes/breakfest/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfest/arabic_omelette.md]: How about we get the review app going and then we could know.
ask @m4v15 he could be your first  reviewer

File: recipes/breakfast/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfast/arabic_omelette.md]: Omelette shouldn't be capitalised

File: recipes/breakfast/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfast/arabic_omelette.md]: Same as above

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: recipes/breakfest/arabic_omelette.md
Comment (finnhodgkin) [reply recipes/breakfest/arabic_omelette.md]: I think maybe @yvonne-liu and @Suuuuha had some of your omelette a few days ago.

File: recipes/breakfast/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfast/arabic_omelette.md]: > I would say flip it when gets golden-brown.

Change to:

I would say flip it when *it* gets golden-brown.

**Review by matthewdking — CHANGES_REQUESTED** [review]

**Comment by m4v15** [comment]
In all seriousness I think we shouldn't be too strict on 100% correct english - we after all don't have a style guide (yet..?) so some things could be very debatable. As long as the message is gotten across I actually think I quite like the small idiosyncrasies of mario's (and others) written english. Like this might be slightly patronising I guess but I think more it would be nicer for each recipe to have the voice of the author within it, with the way that it is written. ❤️

**Comment by Karyum** [comment]
@m4v15 I love what you said but @matthewdking @finnhodgkin are haters

File: recipes/breakfest/arabic_omelette.md
Comment (Suuuuha) [reply recipes/breakfest/arabic_omelette.md]: That's racism :smirk:

File: recipes/breakfest/arabic_omelette.md
Comment (Suuuuha) [reply recipes/breakfest/arabic_omelette.md]: @Karyum since when you trust @m4v15 on Naz recipes??

File: recipes/breakfast/arabic_omelette.md
Comment (Suuuuha) [inline recipes/breakfast/arabic_omelette.md]: change stir to whisk here as well

File: recipes/breakfast/arabic_omelette.md
Comment (Suuuuha) [inline recipes/breakfast/arabic_omelette.md]: pfft.. funny that YOU say so!! you never flipped it right :smirk:

**Review by Suuuuha — CHANGES_REQUESTED** [review]

File: recipes/breakfast/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfast/arabic_omelette.md]: that's such a lie and you know it mate

**Review by m4v15 — APPROVED** [review]

File: recipes/breakfast/arabic_omelette.md
Comment (matthewdking) [inline recipes/breakfast/arabic_omelette.md]: *until

**Review by matthewdking — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

File: recipes/breakfast/arabic_omelette.md
Comment (finnhodgkin) [inline recipes/breakfast/arabic_omelette.md:22]: I've heard that arabic omelettes contain mint. Thoughts?

**Comment by matthewdking** [comment]
@shiryz I believe you have some interesting opinions on this?

**Review by finnhodgkin — APPROVED** [review]
I don't think my comment is a blocker

File: recipes/breakfest/arabic_omelette.md
Comment (Amirk390) [reply recipes/breakfest/arabic_omelette.md]: “the everything” ???

 English 🙃

**Review by matthewdking — APPROVED** [review]

File: recipes/breakfast/arabic_omelette.md
Comment (Karyum) [reply recipes/breakfast/arabic_omelette.md:22]: it's like different accents mate same language but some words change.
and i do say get creative

**Comment by Karyum** [comment]
So we have 2 maintainers approval and 2 people approval who request big change ( *like a lot* ).

This can be merged now

**Comment by m4v15** [comment]
@Karyum ! You can't merge your own PRs! and @Suuuuha has changes requested that she hasn't reviewed yet

**Comment by Karyum** [comment]
@m4v15 no where it states that i can't merge my own PR **NO WHERE** even on master reference it doesn't say. we only explain to students in the course just for good practices with team and having a qa

**Comment by m4v15** [comment]
I mean c'mon, standard practice! the whole place would fall apart without these things! It's not stated anywhere because it's assumed knowledge! Otherwise what's the point in making branches, just commit to master!

**Comment by Karyum** [comment]
@m4v15 Mate 2 maintainer approved 2 people who made lots of changes approved i just mentioned on the reason i skipped @Suuuuha [Click here](#37) 
I have all the rights now to push it back and merge it

**Comment by m4v15** [comment]
Why didn't you just ask suha to re-review it?

**Comment by m4v15** [comment]
You could have said, guys can someone merge it now please otherwise it will take ages - you shouldn't have taken the matters into your own hands

**Pull request merged**
